src/components/select/select.js

```diff
@@ -776,6 +777,14 @@ function OptionDirective($mdButtonInkRipple, $mdUtil) {
     var optionCtrl = ctrls[0];
     var selectCtrl = ctrls[1];
 
+    if (selectCtrl.isMultiple) {
+      element.attr('md-checkbox-enabled', '');
+      if (!checkboxElement) {
+        checkboxElement = angular.element('<div class="_md-container"><div class="_md-icon"></div></div>');
```

Comment:
I believe the behaviour you are describing is, in fact, what is going on here.

Actually if I clone element inside of the `OptionDirective` right before the `return`  statement a new clone will be made every time the `md-option` directive is invoked (if you put a `debugger` on line 760 and navigate to the select demo page, you will notice you hit that `debugger` breakpoint several times).

I have declared the variable at the very top of the file so there should only be one reference present in the code (kept alive in the parent closure), and it is instantiated IFF we have a multi-select (as close to lazy loading as I could manage).

We never clone the element again (if you put a `debugger` on line 782 and navigate to the select demo page, you will notice you hit that `debugger` breakpoint only once).

I agree though that I should rename the variable because my intention was to create a constant. 

I believe putting that constant at the top of the `OptionDirective` initially un-initialized but keeping the if statement would follow more of the "declare variables where they are used" philosophy, but then detecting if the clone has been created already in other `OptionDirective`'s on the page becomes more difficult..